### PR TITLE
Content updates and css fixes

### PR DESCRIPTION
**Pull request by rutuja-google**
Changes proposed in this pull request:

- Content updates and css fixes.
- Updated incorrect values of item for like-icon tag at various places.

image

### CODE REVIEW

File: site/_includes/partials/chrome-100-card-section.njk [hunk 1]
```diff
@@ -188,27 +188,10 @@
             <p class="card-description">One of the gamechangers for web layouts.</p>
             <div class="display-flex gap-top-200 logos">
               {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/PYajnvHnl9nnQpdWDfzA.svg", alt="Promise tag", class="tags gap-top-100", width="120", height="16" %}
-              <like-icon item="100-card-61"></like-icon>
+              <like-icon item="100-card-7"></like-icon>
```
Comment (devnook) [inline site/_includes/partials/chrome-100-card-section.njk:191]: If you change the id of the like icon, you change the asociated likes count. Can you please explainn in the PR description why this change is needed?

**Review by devnook — CHANGES_REQUESTED** [review]

File: site/_includes/partials/chrome-100-card-section.njk (hunk 1 above)
Comment (rutuja-google) [reply site/_includes/partials/chrome-100-card-section.njk:191]: If you check on live it looks like the item values for card mentioned in PR description is 100-card-61 whereas its value should be 100-card-7 as per sequence and placement of cards.

We have received few cards placement changes from Harleen and we have modified their item values as well.

If you are fine to keep the current item values then we can revert back the changes. Please suggest.

Comment (devnook) [reply site/_includes/partials/chrome-100-card-section.njk:191]: The ID is used to associate number of likes with a given card, and this is already running 2 days in the production. It is essential that the IDs (`item` value on `<like-icon>`) stays the same for a given card, doesn't matter where the card is placed in the HTML. Also, if you add a new card, please ensure you give it a new, unique ID (`item` value).

Comment (harleenkbatra08) [reply site/_includes/partials/chrome-100-card-section.njk:191]: We should keep the card id consistent so we don't lose the associated likes count. I don't mind if the number is different based on their placement. Please keep the card id same as before!

Comment (rutuja-google) [reply site/_includes/partials/chrome-100-card-section.njk]: Reverted the item values. Preview build is in process.

**Review by devnook — APPROVED** [review]
